io.github.loot.loot.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.loot.loot
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: LOOT
+modules:
+  - name: application
+    buildsystem: simple
+    build-commands:
+      - install -D bin/LOOT /app/bin/LOOT
+      - install -D lib/libloot.so /app/lib/libloot.so
+      - cp -r share /app
+    sources:
+      - type: archive
+        url: https://github.com/loot/loot/releases/download/0.21.0/loot_0.21.0-Linux.tar.xz
```

Comment:
can't you build from source?

Reply:
It takes 10 minutes and a lot of setup. Flatpak is not the primary target for LOOT, it's been a Windows-only application for a decade, and I'm not about to start maintaining a separate copy of my CI setup just to build flatpaks for Flathub, that's just not worth the effort.

Reply:
then you can build these in the manifest.

Reply:
I would strongly prefer not to, as doing so would make this more of a maintenance burden for me, and would mean that non-Flathub and Flathub Linux builds wouldn't necessarily be the same, which might cause headaches when testing and when dealing with issue reports. I'm also concerned about how I also have no knowledge of or (AFAIK) control over the build environment that Flathub builds run in.

While I was learning how to build Flatpaks I saw many examples of existing packages that don't build from source, so it's clearly not a problem to take the approach I have.

Reply:
> While I was learning how to build Flatpaks I saw many examples of existing packages that don't build from source,

Two wrongs don't make a right.

Reply:
> > While I was learning how to build Flatpaks I saw many examples of existing packages that don't build from source,
> 
> Two wrongs don't make a right.

I'm trying to explain to you why I've taken the approach I have, and this isn't a constructive response, it makes it seem like my PR is not being reviewed in good faith. I've put in a lot of work to get my application working as a Flatpak package, and you're asking me to do more work that will introduce ongoing inconvenience, without any explanation on your part.

This manifest file is a copy of one I've been using to produce Flatpak packages as single-file bundles in CI for the past month or so (and I'll be folding the edits made in response to your review back into the original manifest), so I know the current approach works, and (in addition to the reasons I've already given) I'd rather not introduce discrepancies between Flathub builds and my CI builds.